packages/bot-skeleton/src/services/api/appId.js

```diff
@@ -10,3 +11,111 @@ export const generateDerivApiInstance = () => {
     });
     return deriv_api;
 };
+
+export const getLoginId = () => {
+    return localStorage.getItem('active_loginid');
+};
+
+export const getToken = () => {
+    const active_loginid = getLoginId();
+    const client_accounts = JSON.parse(localStorage.getItem('client.accounts')) || undefined;
+    const active_account = (client_accounts && client_accounts[active_loginid]) || {};
+    return {
+        token: active_account?.token || undefined,
+        account_id: active_loginid || undefined,
+    };
+};
+
+class GetAPI {
```

Comment:
```suggestion
class APIBase {
```
Can we change the class name to APIBase as it's more suitable?

Reply:
Also, can we move this class to a new file with name api-base.js?

Reply:
Sure 🙇 @balakrishna-binary